samples/compression/lz4/src/main.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2020 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <lz4.h>
+
+void main(void)
+{
+	const char *src = "Lorem ipsum dolor sit amet, consectetur adipiscing "
```

Comment:
This might easily overflow the stack so it would be better to make this `static` to avoid that.